Samples/MediaPlayerCS/MainPage.xaml

```diff
@@ -32,26 +32,32 @@
 
             <!-- Control panel -->
             <SplitView.Pane>
-                <StackPanel VerticalAlignment="Stretch" Margin="10, 50, 0, 0">
-                    <TextBlock x:Name="Header" Text="Open Local File or URI"/>
-                    <StackPanel Orientation="Horizontal">
-                        <AppBarButton Content="Open Local File" Icon="OpenFile" Margin="0, 10, 10, 0" Click="OpenLocalFile"/>
-                        <TextBox PlaceholderText="Enter URI and press Enter" InputScope="Url" Width="210" Height="30" KeyUp="URIBoxKeyUp"/>
-                    </StackPanel>
-                    <ToggleSwitch Header="Passthrough Audio MP3" OffContent="OFF" OnContent="ON" IsOn="{x:Bind Mode=TwoWay, Path=Config.PassthroughAudioMP3}"/>
-                    <ToggleSwitch Header="Passthrough Audio AAC" OffContent="OFF" OnContent="ON" IsOn="{x:Bind Mode=TwoWay, Path=Config.PassthroughAudioAAC}" Margin="0,0,0,20" />
+                <ScrollViewer>
+                    <StackPanel VerticalAlignment="Stretch" Margin="10, 50, 0, 0">
+                        <TextBlock x:Name="Header" Text="Open Local File or URI"/>
+                        <StackPanel Orientation="Horizontal">
+                            <AppBarButton Content="Open Local File" Icon="OpenFile" Margin="0, 10, 10, 0" Click="OpenLocalFile"/>
+                            <TextBox PlaceholderText="Enter URI and press Enter" InputScope="Url" Width="210" Height="30" KeyUp="URIBoxKeyUp"/>
+                        </StackPanel>
+                        <ToggleSwitch Header="Auto create playback item" OffContent="OFF" OnContent="ON" IsOn="{x:Bind Mode=TwoWay, Path=AutoCreatePlaybackItem}"/>
```

Comment:
I think we should remove the winAPI only when we've confirmed that ffmpeg can replaces all of its functionality (which I am sure it can, we just need to prove it).
As for the auto create, that was added soley for debugging this solution. We can remove it afterwards.
Regardless, we will still need that scrollviewer in the ham menu, sometimes we get weird UI.